config/constants.go

```diff
@@ -84,6 +85,7 @@ func SetupConstants() {
 	overrideFromEnv(&Constants.CookieDomain, "SERVER_COOKIE_DOMAIN")
 	overrideBoolFromEnv(&Constants.ChatLogsEnabled, "LOG_CHAT")
 	overrideBoolFromEnv(&Constants.ServerMockUp, "PAULING_ENABLE")
+	overrideBoolFromEnv(&Constants.FakeAuthEnabled, "FAKE_AUTH")
 	overrideFromEnv(&Constants.LoginRedirectPath, "SERVER_REDIRECT_PATH")
 	// conditional assignments
```

Comment:
Can you add a log statement here? If FakeAuthEnabled is true, helpers.Logger.Warning("The fake authentication flag has been set. Anyone can log in as any user.")